build.gradle.kts

```diff
@@ -0,0 +1,162 @@
+import java.util.Date
+
+plugins {
+    kotlin("multiplatform") version "1.3.41"
+    id("com.jfrog.bintray") version "1.8.4"
+    `maven-publish`
+    `java-library` //todo: ask whether we need javadoc at all as it is always empty
```

Comment:
Do you need javadocs? The jars are empty

Reply:
If you check this you'll see jars are not empty: https://repo1.maven.org/maven2/io/github/microutils/kotlin-logging/1.6.26/

It is required by maven central

Reply:
I think [dokka](https://github.com/Kotlin/dokka) would help

Reply:
Sorry, by empty I meant jar contains only MANIFEST.MF

Reply:
But there are no javadocs themselves